tests/integrations/test_burr_pydantic.py

```diff
@@ -110,8 +110,7 @@ class MyModelWithConfig(pydantic.BaseModel):
         foo: int
         arbitrary: Arbitrary
 
-        class Config:
-            arbitrary_types_allowed = True
+        model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
no pydantic version assumption to make?

Reply:
Nope, this is available on the later 1.X versions. 0.X won't work but. I think that's OK.

Reply:
`model_validate` is though so I'll add